dist/fabric.js

```diff
@@ -14602,6 +14626,7 @@ fabric.util.object.extend(fabric.StaticCanvas.prototype, /** @lends fabric.Stati
      * Sets "color" of an instance (alias of `set('fill', &hellip;)`)
      * @param {String} color Color value
      * @return {fabric.Object} thisArg
+     * @deprecated since 3.5.0
```

Comment:
i need to release a 3.5.0 just for those deprecations so that i can remove them in 4.0